{"uuid":"5f7cd9b8-1f9d-4542-a02f-885e9770e87a","historyId":"9b63cd94f88bbcd5de3a7f5dc3124c4b","fullName":"Array: The user is presented with error message for code with invalid syntax in tryEditor for Search the array link","labels":[{"name":"tag","value":"arrays"},{"name":"host","value":"LAPTOP-S7F1L181"},{"name":"thread","value":"10368@LAPTOP-S7F1L181.main(1)"},{"name":"feature","value":"Array"},{"name":"story","value":"The user is presented with error message for code with invalid syntax in tryEditor for Search the array link"},{"name":"suite","value":"Array"},{"name":"testClass","value":"The user is presented with error message for code with invalid syntax in tryEditor for Search the array link"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/xunee/eclipse-workspace-2022/DS-ALGO-/src/test/resources/features/5arrays.feature"},{"name":"package","value":"src.test.resources.features.5arrays_feature.Array"}],"links":[],"name":"The user is presented with error message for code with invalid syntax in tryEditor for Search the array link","status":"passed","stage":"finished","steps":[{"name":"Given  The user is on \"Question\" page of \"Search the array\" after logged in","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673371475318,"stop":1673371475478},{"name":"When  The user enter python code with invalid syntax in tryEditor from sheet \"pythonCode\" and 1","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673371475478,"stop":1673371475779},{"name":"And  The user clicks on run button","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673371475779,"stop":1673371475922},{"name":"Then  The user should be presented with error message as \"SyntaxError: bad input on line 1\"","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673371475923,"stop":1673371475940}],"attachments":[],"parameters":[{"name":"Sheetname","value":"pythonCode"},{"name":"RowNumber","value":"1"}],"start":1673371475318,"stop":1673371475940}